LR_Tourist/TouristWebApp/Models/ChangeRoleViewModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.AspNetCore.Identity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace TouristWebApp.Models
+{
+    public class ChangeRoleViewModel
+    {
+        public string UserId { get; set; }
+        public string UserEmail { get; set; }
+        public List<IdentityRole> AllRoles { get; set; }
+        public IList<string> UserRoles { get; set; }
+        public ChangeRoleViewModel()
+        {
+            AllRoles = new List<IdentityRole>();
```

Comment:
From my perspective, it is useless constructor.

Reply:
fixed